localization/localization_node/src/localization.cc

```diff
@@ -27,55 +27,60 @@
 
 namespace localization_node {
 
-Localizer::Localizer(sparse_mapping::SparseMap* comp_map_ptr) :
-      map_(comp_map_ptr) {
-}
-
-Localizer::~Localizer(void) {
-}
+Localizer::Localizer(sparse_mapping::SparseMap* map): map_(map) {}
+
+void Localizer::ReadParams(config_reader::ConfigReader& config) {
+  camera::CameraParameters cam_params(&config, "nav_cam");
+  std::string prefix;
+  const auto detector_name = map_->GetDetectorName();
+  if (detector_name == "ORGBRISK") {
+    prefix = "brisk_";
+  } else if (detector_name == "TEBLID") {
+    prefix = "teblid_";
```

Comment:
(Possibly support "TEBLID512" and "TEBLID256"?)

Reply:
Done